tests/api/test_devicespec.py

```diff
@@ -14,21 +14,20 @@
 """
 Unit tests for strawberryfields.api.devicespec
 """
+import inspect
+import regex as re
+
 import pytest
-import textwrap
 
 from strawberryfields.api import DeviceSpec
 from strawberryfields.compilers import Ranges
-import strawberryfields as sf
-
-import blackbird
 
-# pylint: disable=bad-continuation,no-self-use,pointless-statement
+# pylints: disable=bad-continuation,no-self-use,pointless-statement
```

Comment:
What's the difference between `# pylint` and `# pylints`?

Reply:
Oh, whoops, I just added the `s` to check whether these actually needed to be disabled.